Sources/Core/Filters/Inline/InlineFilterView.swift

```diff
@@ -5,26 +5,20 @@
 import UIKit
 
 protocol InlineFilterViewDelegate: class {
-    func inlineFilterView(_ inlineFilterView: InlineFilterView, didChangeSegment segment: Segment, at index: Int)
+    func inlineFilterView(_ inlineFilteView: InlineFilterView, didChange segment: Segment, at index: Int)
     func inlineFilterView(_ inlineFilterview: InlineFilterView, didTapExpandableSegment segment: Segment)
 }
 
 final class InlineFilterView: UIView {
 
     // MARK: - Public Properties
 
-    var vertical: String?
-
-    var segmentTitles: [[String]] = [] {
-        didSet {
-            setupItems()
-        }
-    }
-
     weak var delegate: InlineFilterViewDelegate?
 
     // MARK: - Private properties
 
+    private var vertical: String?
```

Comment:
I think we can remove those two properties and merge `setupItems` with `configure` method where we have all the data needed to build segments.

Reply:
Good idea! :)